stubs/redis/@tests/stubtest_allowlist.txt

```diff
@@ -19,6 +19,12 @@ redis.typing.CommandsProtocol.__init__
 # unclear problems
 redis.Sentinel.master_for
 redis.Sentinel.slave_for
+redis.asyncio.client.Pipeline.command_info
+redis.asyncio.client.Pipeline.debug_segfault
+redis.asyncio.client.Pipeline.memory_doctor
+redis.asyncio.client.Pipeline.memory_help
+redis.asyncio.client.Pipeline.script_debug
+redis.asyncio.client.Pipeline.shutdown
```

Comment:
These allowlist entries are all because the method is an `async def` method at runtime but not an `async def` method in the stub.

1. Are you sure it's correct to have them as not being `async def` in the stub?
2. If you are sure, can you put these allowlist entries in a separate section, and add a comment explaining why we're ignoring the stubtest errors? I don't think these entries belong in the "unclear problems" section, since we know why stubtest is complaining here.

Reply:
Hi @AlexWaygood 

I'm sure in the sense that it doesn't really matter, in principle this function should be syncronous in the pipeline, just like all other redis commands. In practice, because the functions are defined as `async` in the superclass unlike most commands (I can't really tell why), then any attempt to use these commands in a pipeline would result in a bug.

I will place these entreis in their own section